internal/utils/flags/flags.go

```diff
@@ -107,7 +108,14 @@ func (esv *EnumSetValue) validateAndRemoveDuplicates() error {
 }
 
 func (esv *EnumSetValue) errInvalidEnumValue() error {
-	return fmt.Errorf(`unsupported value, use one of ["%s"] instead`, strings.Join(esv.validValues, `", "`))
+	bSep := []byte(`", "`)
+	out := make([]byte, 0, (1+len(bSep))*len(esv.validValues))
```

Comment:
same thing about removing the indexing logic here and just use a `strings.Builder` for concatenation here